examples/printimages.py

```diff
@@ -25,22 +25,76 @@
 author = 'jmpr22'
 import sys
 
-from glancesync import glancesync_fi
+from glancesync.glancesync import GlanceSync
+
+def _printimages(imagesregion, comparewith=None):
+    """ print a report about the images present on the specified region
+
+    See the documentation of GlanceSync.printimages for more details
+
+    :param imagesregion: the region of print
+    :param comparewith: the master region dictionary, used to compute the
+              image synchronization status.
+    :return: this function doesn't return anything.
+    """
+
+    images = list(
+        image for image in imagesregion if image.is_public and
+        ('nid' in image.user_properties and 'type' in image.user_properties))
+    images.sort(key=lambda image: image.user_properties['type'] + image.name)
+    properties = ('type', 'nid')
+    for image in images:
```

Comment:
from line 46 to line 53 are code block that could be put like a function in order that you do not replicate code...